File: js/primes.js
function printPrimes(limit) {
    const primes = new Array(limit+1).fill(true);
  
    // 0 and 1 are not prime
    primes[0] = false;
    primes[1] = false;
  
    for (let i = 2; i <= limit; i++) {
      if (primes[i]) {
        console.log(i);
        let j = i * i;
        while (j <= limit) {
          primes[j] = false;
          j += i;
        }
      }
    }
  }
  
  printPrimes(100000000);
  

File: python/primes.py
def print_primes(limit):
    primes = [True] * (limit+1)

    # 0 and 1 are not prime
    primes[0] = False
    primes[1] = False

    for i in range(2, limit+1):
        if primes[i]:
            print(i)
            j = i * i
            while j <= limit:
                primes[j] = False
                j += i

print_primes(100000000)
